api/cpp/cbindgen.rs

```diff
@@ -75,6 +75,7 @@ fn default_config() -> cbindgen::Config {
         defines: [
             ("target_pointer_width = 64".into(), "SLINT_TARGET_64".into()),
             ("target_pointer_width = 32".into(), "SLINT_TARGET_32".into()),
+            ("target_arch = wasm32".into(), "SLINT_CPP_NEVER_WASM".into()), // Disable any wasm guarded code in C++, too - so that there are no gaps in enums.
```

Comment:
```suggestion
            ("target_arch = wasm32".into(), "SLINT_TARGET_WASM".into()), // Disable any wasm guarded code in C++, too - so that there are no gaps in enums.
```

CPP_NEVER_WASM will look wrong and reverse when looking at the generated code

Reply:
Good point :)